app/views/pageflow/entries/index.atom.builder

```diff
@@ -0,0 +1,32 @@
+atom_feed do |feed|
+  feed.title @account.name
+  feed.url pretty_theming_url(@theming)
+  feed.updated @entries.first.updated_at if @entries.present?
+  feed.favicon image_path("pageflow/themes/#{@theming.theme.directory_name}/favicon.ico")
+
+  # we can have next_url once we can upgrade ActiveAdmin->Kaminari
+  # feed.next_url pretty_theming_url(path_to_next_page(@entries))
+
+  @entries.each do |pageflow_entry|
+    published_entry = Pageflow::PublishedEntry.new(pageflow_entry)
```

Comment:
I'd suggest moving all of the entry query building logic that lives in the controller into a separate query class (something like `FeedEntriesQuery`). This gives us a nice place to test that only the desired entries show up in the feed. We can then also wrap all of the entries in `PublishedEntriy` decorators right there to further simplify this view code.